builder/amazon/common/step_create_tags.go

```diff
@@ -130,14 +165,25 @@ func (s *StepCreateTags) Cleanup(state multistep.StateBag) {
 	// No cleanup...
 }
 
-func ConvertToEC2Tags(tags map[string]string) []*ec2.Tag {
-	var ec2tags []*ec2.Tag
+func ConvertToEC2Tags(tags map[string]string, region, sourceAmiId string, ctx interpolate.Context, ui packer.Ui) ([]*ec2.Tag, error) {
+	var amiTags []*ec2.Tag
 	for key, value := range tags {
-		log.Printf("[DEBUG] Creating tag %s=%s", key, value)
-		ec2tags = append(ec2tags, &ec2.Tag{
+
+		ctx.Data = &BuildInfoTemplate{
+			SourceAMI:   sourceAmiId,
+			BuildRegion: region,
+		}
+		interpolatedValue, err := interpolate.Render(value, &ctx)
+		if err != nil {
+			return amiTags, fmt.Errorf("Error processing tag: %s:%s - %s", key, value, err)
+		}
+
+		ui.Message(fmt.Sprintf("Adding tag: \"%s\": \"%s\"", key, interpolatedValue))
```

Comment:
I think the creating tags info is less useful as a UI message and more useful for debugging. I'm not set on this, but I think I would prefer the tag message as a debug log